### DISCUSSION TITLE
YI Home Camera 1080p 4FCN

### DISCUSSION THREAD

**Discussion by yevgeniy1970** [roleoroleo/yi-hack-MStar#529] [Q&A]
YI Home Camera 1080p 4FCN MSC313E after flashing the firmware it clicks twice and blinks constantly, what should I do?

**Comment by roleoroleo** [comment]
Try to downgrade to a previous version.
It should restore the cam.

**Comment by yevgeniy1970** [comment]
Подключил к wifi загорелась синим но молчит нет голоса

чт, 28 дек. 2023 г., 23:11 roleo ***@***.***>:

**Comment by yevgeniy1970** [comment]
Заработала спасибо огромное

**Comment by yevgeniy1970** [comment]
Но голос пропал

**Reply by roleoroleo** [reply]
It's normal.
